CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ cmake_minimum_required(VERSION 3.4)
 
 include(ExternalProject)
 
-project( CCTag )
+project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
```

Comment:
@fabiencastan @zvrba @griwodz @lcalvet 
Two things:
* if you agree we can express the lib version with `project()`, this automatically sets all the variables to recover the major, minor, patch numbers (see https://cmake.org/cmake/help/v3.0/command/project.html)
* More importantly I set here 1.0.0 as a first version of the library. It's arbitrary but it seemed to me a good number to start with :-). Anyway, just decide a number if you prefer another, the important thing is that what we decide here will be the minimal version supported in all the other libs, openMVG and other LABO libs.